rpcs3/Emu/Cell/SPUThread.cpp

```diff
@@ -4535,7 +4535,7 @@ u32 spu_thread::get_ch_count(u32 ch)
 	case SPU_RdSigNotify1:    return ch_snr1.get_count();
 	case SPU_RdSigNotify2:    return ch_snr2.get_count();
 	case MFC_RdAtomicStat:    return ch_atomic_stat.get_count();
-	case SPU_RdEventStat:     return get_events().count;
+	case SPU_RdEventStat:     return ::narrow<u32>(get_events().count);
```

Comment:
static cast here, this for performance reasons and count cannot be above 1 anyway here. (because this is spu interpreter code)